doc/pattern_tools/svgfig.py

```diff
@@ -449,12 +449,9 @@ def __standalone_xml(self, indent, newl):
 
         return output
 
-    def interpret_fileName(self, fileName=None):
-        if fileName is None:
-            fileName = _default_fileName
-        if re.search("windows", platform.system(), re.I) and not os.path.isabs(fileName):
-            fileName = _default_directory + os.sep + fileName
```

Comment:
`_default_directory` is not used any more. So it should be removed with initialization code.

Reply:
Nice catch. I removed it.